packages/react-link/src/components/Link/Link.types.ts

```diff
@@ -65,9 +74,6 @@ export type LinkProps = ComponentPropsCompat &
     secondary?: boolean;
   };
 
-/**
- * {@docCategory Link}
- */
-export interface LinkState extends LinkProps {
-  ref: React.Ref<HTMLElement>;
-}
+export interface LinkProps extends Omit<ComponentProps<LinkSlots>, 'onClick'>, Partial<LinkCommons> {}
+
+export interface LinkState extends ComponentState<LinkSlots>, LinkProps {}
```

Comment:
```suggestion
export interface LinkState extends ComponentState<LinkSlots>, LinkCommons {}
```

This should use `LinkCommons`

Reply:
Noted and changed. Thanks!